compiler/src/iree/compiler/Dialect/Flow/IR/test/dispatch_ops.mlir

```diff
@@ -111,3 +111,37 @@ func.func @inplaceTypeChange(%arg0: tensor<4x?xf32>) -> tensor<?x4xf32> {
   %0 = flow.dispatch @ex0::@dispatch_fn[%cst](%arg0) : (tensor<4x?xf32>{%dim0}) -> %arg0 as tensor<?x4xf32>{%dim0}
   return %0 : tensor<?x4xf32>
 }
+
+// -----
+
+// CHECK-LABEL: @region
+// CHECK-SAME: (%[[ARG0:.+]]: tensor<?x?xf32>)
+func.func @region(%arg0: tensor<?x?xf32>) -> tensor<?x?xf32> {
+  // CHECK: %[[R:.*]] = flow.dispatch.region -> (tensor<?x?xf32>{%{{.*}}, %{{.*}}}) {
```

Comment:
The return type here instead of after the region is strange, but I am guessing this is to be consistent with `flow.dispatch.workgroups`. So not holding for it, but just a nit in case you want to change it.

Reply:
Yes, this is for consistency with `workgroups`. The alternative would be to print it like a normal operand (`dispatch.region(%0, %1)`), but then it is a bit harder to figure out which values belong to which result types.